core/controllers/translator_test.py

```diff
@@ -72,7 +72,42 @@ def setUp(self):
         response = self.get_html_response('/create/%s' % self.EXP_ID)
         self.csrf_token = self.get_csrf_token_from_response(response)
 
-    def test_transator_can_save_valid_change_list(self):
+    def test_put_with_no_payload_version_raises_error(self):
+        state_name = feconf.DEFAULT_INIT_STATE_NAME
```

Comment:
Here and below, might as well inline this since you're only using it once

Reply:
Done